src/main/java/com/lorepo/icplayer/client/PlayerEntryPoint.java

```diff
@@ -378,6 +380,21 @@ private JavaScriptObject getScoreWithMetadata() {
 		return jsScores;
 	}
 
+	private void clearContentData() {
+		pageLoadedListener = null;
+		externalEventListener = null;
+		pageScrollToListener = null;
+		statusChangedListener = null;
+		outstretchHeightListener = null;
+		contextMetadata = null;
+		externalVariables = null;
+		clearGlobalAddonVariables();
+	}
+
+	private native void clearGlobalAddonVariables()/*-{
+		$wnd.savedPanel = null; //IWB_Toolbar
```

Comment:
Jeżeli ta metoda się rozszerzy to proponuję, aby była metoda per addon np.  clearGlobalIWBToolbarVariables